src/include/cache.h

```diff
@@ -198,16 +198,18 @@ struct __wt_cache {
     uint32_t las_fileid;       /* Lookaside table file ID */
     uint64_t las_insert_count; /* Count of inserts to lookaside */
     uint64_t las_remove_count; /* Count of removes from lookaside */
-    uint64_t las_pageid;       /* Lookaside table page ID counter */
+    uint64_t las_counter;      /* Lookaside counter */
```

Comment:
It wasn't alphabetised before. Do you want me to slot it in between `las_insert_count` and `las_remove_count`?

Personally I think it makes more sense for related things to be next to each other rather than just alphabetising everything.